### DISCUSSION TITLE
Which mapper to use

### DISCUSSION THREAD

**Discussion by JXAUDZF** [ConesaLab/SQANTI3#265] [Q&A] [answered]
seem to the long reads can not used to star, therefore how should I do to solve the problem?
![Uploading image.png…]()

**Comment by JXAUDZF** [comment]
<img width="1312" alt="image" src="https://github.com/ConesaLab/SQANTI3/assets/115453641/a2a9682c-9964-48ab-b1c4-a4b1fbdbcd34">

**Comment by carolinamonzo** [comment, marked as answer]
Hi @JXAUDZF, there are mappers made for mapping long reads. For example, SQANTI uses minimap2, but there are others such as uLTRA.
